tests/ted/test.py

```diff
@@ -164,6 +171,8 @@ def generate_cases(self):
 
         for vals in itertools.product(*values):
             case = collections.OrderedDict(zip(keys, vals))
+            case['device'] = self.device
```

Comment:
well, I don't understand how you pass selected device to the apps... I expect to see -device parameter added to sample command lines and I can't find that...

Reply:
This happens in `def other_options` run.py:108
So `-device /dev/dri/renderD128` flag appears in result parameters for sample.

Reply:
ok